google/cloud/security/common/gcp_api/admin_directory.py

```diff
@@ -0,0 +1,83 @@
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Wrapper for Admin Directory  API client."""
+
+from googleapiclient.errors import HttpError
+from httplib2 import HttpLib2Error
+from ratelimiter import RateLimiter
+
+from google.cloud.security.common.gcp_api._base_client import _BaseClient
+from google.cloud.security.common.gcp_api._base_client import ApiExecutionError
+from google.cloud.security.common.util.log_util import LogUtil
+
+
+LOGGER = LogUtil.setup_logging(__name__)
+
+DEFAULT_MAX_QUERIES = 150000
+DEFAULT_RATE_BUCKET_SECONDS = 86400
+
+
+class AdminDirectoryClient(_BaseClient):
+    """GSuite Admin Directory API Client."""
+
+    API_NAME = 'admin'
+
+    def __init__(self, credentials=None, rate_limiter=None):
+        super(AdminDirectoryClient, self).__init__(
+            credentials=credentials, api_name=self.API_NAME)
+        if rate_limiter:
+            self.rate_limiter = rate_limiter
+        else:
+            self.rate_limiter = self.get_rate_limiter()
+
+    @staticmethod
+    def get_rate_limiter():
+        """Return an appriopriate rate limiter."""
```

Comment:
It seemed unimportant for the method to know about it's class. It's placement here was truly for organization reasons as it seemed to be the best spot.